content/en/serverless/troubleshooting/_index.md

```diff
@@ -38,35 +38,51 @@ To ensure the actual changes made to your Lambda functions are expected, try set
+Datadog can collect metrics and resource tags from AWS through an [integration with AWS][10] (optional). If you are missing certain metrics from CloudWatch and Lambda resource tags, ensure that the AWS integration is properly configured.
 
 ## Ensure the tags are configured
 
 If you have trouble applying the Datadog standard `service`, `env`, and `version` tags to the collected data, make sure the environment variables `DD_SERVICE`, `DD_ENV`, and `DD_VERSION` are configured on your Lambda functions. For custom tags, ensure `DD_TAGS` is configured.
 
 If you want to enrich the data collected with your AWS Lambda resource tags, make sure the [Datadog integration for AWS][10] is properly configured.
 
-## Get help
+## Get help with Serverless Flare
 
 For quick questions, post in the _#serverless_ channel of the [Datadog Slack community][11].
 
-If you have followed all the troubleshooting steps above and want help from [Datadog Support][12], include the following information in your ticket:
+If you have followed all the troubleshooting steps above and want help from [Datadog Support][12], use Serverless Flare to send important configuration info to support.
 
-1. Basic information about your Lambda function:ARN, runtime, handler, layers, environment variables, and tags. Focus on one function first if you have the same issue with many.
-1. If the Lambda function is configured to send data through logs using the Datadog Forwarder Lambda function, include basic information about the Forwarder Lambda function, as well as the subscription filters configured on your Lambda function's log group.
-2. The installation method you followed, such as _Serverless Framework_ or _AWS CDK_.
-3. The alternative installation method you attempted, such as _Datadog CLI_ or _Custom_.
-4. Debugging logs from your own Lambda function.
-5. Debugging logs from the Datadog Forwarder Lambda function (if used).
-6. The project configuration files, with **redacted hardcoded secrets**, such as `serverless.yaml`, `package.json`, `package-lock.json`, `yarn.lock`, `tsconfig.json`, and `webpack.config.json`.
+1. Create a [Zendesk ticket](https://help.datadoghq.com/hc/en-us/requests/new)
+2. Download the latest version of the [Datadog CLI](https://github.com/DataDog/datadog-ci/#how-to-install-the-cli)
+
+    ```sh
+    npm install -g @datadog/datadog-ci
+    ```
+
+3. Use the serverless flare command to automatically collect and submit data about your Lambda function to Datadog Support.
+
+    ```sh
+    datadog-ci lambda flare
+    ```
+
+This command will attatch to following info to your Zendesk case:
+1. Basic information about your Lambda function: ARN, runtime, handler, layers, environment variables, and tags. Focus on one function first if you have the same issue with many.
```

Comment:
Straight copy paste. This is meant if users send the data manually. Either you change this to be in its own section of manual data gathering, or you remove it.